scripts/rp.py

```diff
@@ -400,8 +402,19 @@ def process(self, p, active, a_debug , rp_selected_tab, mmode, xmode, pmode, ara
         self.slowlora = OPTUSEL in options
 
         if type(polymask) == str:
+            image = None
             try:
-                polymask,_,_ = draw_image(np.array(Image.open(polymask)))
+                image = draw_image(np.array(Image.open(polymask)))
+            except:
+                if polymask.startswith("data:image/"):
+                    polymask = polymask.split(";")[1].split(",")[1]

```

Comment:
maxsplit=1 could be added, for better oerformance
like .split(";", maxsplit=1)

Reply:
improved